__tests__/habits/habits.test.js

```diff
@@ -135,16 +134,39 @@ describe('API - Habit', () => {
 					done();
 				}));
 
+		test('Logging the habit of another user should be unauthorized', done => {
+			// Create another user
+			userData.email = 'test2@test.cc';
+			let token2;
+			request
+				.post('/api/auth/register')
+				.send(userData)
+				.end((err, { body }) => {
+					body.id = body._id;
+					token2 = createToken(body, process.env.JWT_SECRET, '1h');
+					// Log habit
+					request
+						.patch(`/api/habit/${habitId}/log`)
+						.set('Authorization', token2)
+						.end((error, res) => {
+							if (error) throw err;
+							expect(res.status).toBe(401);
+							expect(res.body.message).toBe('Unauthorized');
+							done();
+						});
+				});
+		});
+
 		test('If a habit is logged without a time, the current time should be logged', done =>
 			request
 				.patch(`/api/habit/${habitId}/log`)
 				.set('Authorization', token)
 				.end((err, res) => {
 					if (err) throw err;
 					expect(res.status).toBe(200);
-					expect(res.body.log.length).toBe(1);
+					expect(res.body.updated.log.length).toBe(1);
```

Comment:
NOTE: I've noticed this in Samuels PR, it's not a critical issue, but this shouldn't be called `updated` as it's not that intuitive, habit would be more straightforward. I'll fix this in the next PR, but if you open PR before mine feel free to fix it yourself.